src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -258,11 +264,16 @@ const ComposeForm = props => {
 
   useEffect(
     () => {
-      if (messageInvalid) {
+      if (
+        messageInvalid || 
+  (isSignatureRequired && (signatureInvalid || checkboxInvalid))
+        (isSignatureRequired && signatureInvalid) ||
+        (isSignatureRequired && checkboxInvalid)
```

Comment:
looks like these linter warnings need to be addressed. You may want to check your linter locally